react-components/src/queries/__test__/validateQueries.test.js

```diff
@@ -31,6 +31,7 @@ describe.each([
     });
 
     expect(files.length).toBeGreaterThan(0); // Ensures we read the right folder
+    // eslint-disable-next-line no-console
     console.log(`Validating ${files.length} GraphQL requests against Magento schema ${version}`);
```

Comment:
Please remove this line instead of setting an `eslint` exception.

Reply:
It was really useful to have that line in the tests, to see at least that something happens and that multiple versions are being processed.

Reply:
@cjelger the Magento version information is already displayed as part of the test description